lib/chef/knife/core/windows_bootstrap_context.rb

```diff
@@ -238,6 +256,10 @@ def local_download_path
           local_download_path = "%TEMP%\\chef-client-latest.msi"
         end
 
+        def msi_url
```

Comment:
It seems like some of the logic could be moved in here from windows-chef-client-msi.erb, so if anyone else uses a custom template and needs our URL stuff they wouldn't be hardcoding it?